platform/darwin/src/MGLSDKMetricsManager.m

```diff
@@ -18,6 +18,21 @@
     return eventName;
 }
 
+// Taken verbatim from NXFreeArchInfo header documentation
```

Comment:
This code should be wrapped around:
`#if TARGET_OS_IOS`

Reply:
Great point!